get_trust_anchor.py

```diff
@@ -344,13 +325,18 @@ def get_matching_ksk(KSKRecords, ValidTrustAnchors):
     return MatchedKSKs
 
 
-def export_ksk(ValidKSKs, DSRecordFileName, DNSKEYRecordFileName):
```

Comment:
Yes, correct. If we want the functions to possibly be re-used, the filenames should be in the function call. Please revert this back.